src/database/config/database.js

```diff
@@ -7,12 +7,24 @@ config();
 
 const env = envConfigs[process.env.NODE_ENV] || envConfigs.development;
 
+// const dbase = new Sequelize(env.url, {
+//   dialectOptions: {
+//     ssl: {
+//       require: true,
+//       rejectUnauthorized: false,
+//     },
+//   },
+// });
+
 const dbase = new Sequelize(env.url, {
-  dialectOptions: {
-    ssl: {
-      require: true,
-      rejectUnauthorized: false,
-    },
+  host: 'localhost',
```

Comment:
what's the use of this property??
Aren't we using the host from the connection URL string